lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3568,6 +3589,24 @@ namespace Js
         return (flags != ImplicitCall_None) || arrayIteratorPrototypeNext != scriptContext->GetLibrary()->GetArrayIteratorPrototypeBuiltinNextFunction();
     }
 
+    bool JavascriptLibrary::InitializeBigIntConstructor(DynamicObject* bigIntConstructor, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode)
+    {
+        typeHandler->Convert(bigIntConstructor, mode, 17);//what is 17???
```

Comment:
17 is how many slots to pre-allocate for properties. It's a minor performance optimization to pre-allocate the right number if you happen to know it, because they can be allocated inline with the object rather than in a separate array. In this case it looks like you want 3.